command/acl/authmethod/update/authmethod_update_test.go

```diff
@@ -726,3 +742,28 @@ func TestAuthMethodUpdateCommand_k8s_noMerge(t *testing.T) {
 		require.Equal(t, acl.TestKubernetesJWT_B, config.ServiceAccountJWT)
 	})
 }
+
+func getTestMethod(t *testing.T, client *api.Client, methodName string) *api.ACLAuthMethod {
+	t.Helper()
+
+	method, _, err := client.ACL().AuthMethodRead(
+		methodName,
+		&api.QueryOptions{Token: "root"},
+	)
+	require.NoError(t, err)
+	require.NotNil(t, method)
+
+	// zero these out since we don't really care
+	method.CreateIndex = 0
```

Comment:
Why are these here?

Reply:
Because it's hard to use `require.Equals` if you don't know what these two field values are set to. For all of these tests I'm just setting these to  zero so I can test the rest of the fields round tripped easily.

Reply:
o til thank you :)